plugins/inlinetoolbar/skins/kama/inlinetoolbar.css

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+.cke_balloon.cke_inlinetoolbar
+{
+	background: #fff;
+	border: 1px solid #D3D3D3;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_content
+{
+	min-height: 10px;
+}
+
+/* side: [ bottom, top ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_top
+{
+	border-color: #D3D3D3 transparent;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top
+{
+	left: -10px;
+}
+
+/* side: [ bottom ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_bottom
+{
+	border-width: 10px 10px 0;
+	bottom: -10px;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer > .cke_balloon_triangle_inner.cke_balloon_triangle_bottom
```

Comment:
Again (I asked for this before) please relax the CSS selectors. I'm sure this one will be good with `.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer .cke_balloon_triangle_inner.cke_balloon_triangle_bottom`, possibly even with `.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom` (just assuming, don't have the branch checked out for testing).

Hell, looking at original balloonpanel selectors, it could be even possible to simplify it down to `.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom`.

Reply:
Talking about specificity, see https://www.smashingmagazine.com/2007/07/css-specificity-things-you-should-know/#specificity-basic-principles in a spare time.